app/src/main/res/values/strings.xml

```diff
@@ -113,23 +114,23 @@
     <string name="sharedpreference_sort" translatable="false">sharedpreference_sort</string>
     <string name="sharedpreference_sort_order" translatable="false">sharedpreference_sort_order</string>
     <string name="sharedpreference_sort_direction" translatable="false">sharedpreference_sort_direction</string>
-    <string name="intent_import_card_from_url_share_text">I want to share a card with you</string>
+    <string name="intent_import_card_from_url_share_text">I want to share something from Catima with you</string>
     <string name="intent_import_card_from_url_host_catima_app" translatable="false">catima.app</string>
     <string name="intent_import_card_from_url_path_prefix_catima_app" translatable="false">/share</string>
     <string name="intent_import_card_from_url_host_thelastproject" translatable="false">thelastproject.github.io</string>
     <string name="intent_import_card_from_url_path_prefix_thelastproject" translatable="false">/Catima/share</string>
     <string name="intent_import_card_from_url_host_brarcher" translatable="false">brarcher.github.io</string>
     <string name="intent_import_card_from_url_path_prefix_brarcher" translatable="false">/loyalty-card-locker/share</string>
-    <string name="importSuccessful">Card data imported</string>
-    <string name="exportSuccessful">Card data exported</string>
+    <string name="importSuccessful">Data imported</string>
+    <string name="exportSuccessful">Data exported</string>
     <string name="enter_group_name">Enter group name</string>
     <string name="groups">Groups</string>
     <string name="group_edit">Edit Group</string>
     <string name="noGroups">Click the + plus button to add groups for categorization first.</string>
-    <string name="noGroupCards">This group does not contain any cards</string>
+    <string name="noGroupCards">This group is empty</string>
     <plurals name="groupCardCount">
-        <item quantity="one"><xliff:g>%d</xliff:g> card</item>
-        <item quantity="other"><xliff:g>%d</xliff:g> cards</item>
+        <item quantity="one"><xliff:g>%d</xliff:g></item>
+        <item quantity="other"><xliff:g>%d</xliff:g></item>
```

Comment:
![image](https://user-images.githubusercontent.com/1885159/144679915-e8b3408d-ccb2-48aa-b24d-282a317578f8.png)

You're changing "2 cards" to "2" here. Will look awful and be very confusing for users to see a number with no explanation.

Reply:
Maybe it could be a visual representation?  [barcodeemoji] x 2  [coupon] x3   [?] x 4
Or [barcodeemoji][barcodeemoji][coupon][coupon][coupon][?][?][?][?]

Reply:
That could be possible maybe, assuming we can tell Android to use the old text for screen readers. But at that point we just turn it into 2 strings to manage.